inst/@sym/abs.m

```diff
@@ -40,7 +40,7 @@
   if (nargin ~= 1)
     print_usage ();
   end
-  y = uniop_helper (x, 'Abs');
+  y = op_helper ('Abs', x);
```

Comment:
For many of these files, the change needs to be made in `generate_functions.py` (see note in the help text).